weather-cal.php

```diff
@@ -92,6 +92,8 @@ function makeDescriptions($data) {
 function displayTemp($temp, $display) {
   if ($display == 'day') {
     return round($temp['day']) . '°';
+  } else if ($display == 'high-low'){
```

Comment:
Should [README.md](https://github.com/vejnoe/weather-in-your-calendar/blob/master/README.md) be updated with the new option?

Reply:
Oh, good point, it probably should.

I'm pretty new to github, so what would be the best way to do so? Can add the change to this pull request, or would I open a new one to fix the README.md (or would you or someone else just fix it in vejnoe:master after the fact)?

Reply:
nvm, figured it out. Resolving this conversation and added new commit to pull request.